compiler/src/iree/compiler/GlobalOptimization/GeneralizeLinalgNamedOps.cpp

```diff
@@ -41,17 +44,16 @@ static bool isConvFoldableToContraction(linalg::LinalgOp linalgOp) {
 
   if (!llvm::all_of(convDims.strides,
                     [](int64_t element) { return element == 1; })) {
+    LLVM_DEBUG(DBGS() << "conv not foldable: non-unit strides\n");
```

Comment:
You can declare a macro for the case that we need a `\n` at the end. We have this style in IREE and MLIR codebase.

```cpp
#define LDBG(X) LLVM_DEBUG(DBGS() << X << "\n")

LDBG("conv not foldable: non-unit strides");
```